File: doc/codes/python/examples/api/basics/sampling-config.py
from autd3 import Duration
from autd3.units import kHz
from autd3.value import Nearest, SamplingConfig

# ANCHOR: api
SamplingConfig(10)
SamplingConfig(4.0 * kHz)
SamplingConfig(Duration.from_micros(250))
SamplingConfig(Nearest(4.0 * kHz))
SamplingConfig(Nearest(Duration.from_micros(250)))
# ANCHOR_END: api

File: doc/codes/csharp/examples/api/basics/sampling-config.cs
using System;
using AUTD3;
using static AUTD3.Units;

namespace DocSamples.ApiBasicsSamplingConfig;

internal static class Sample
{
    internal static void Run()
    {
        // ANCHOR: api
        new SamplingConfig(10);
        new SamplingConfig(4.0f * kHz);
        new SamplingConfig(TimeSpan.FromMicroseconds(250));
        new SamplingConfig(Nearest(4.0f * kHz));
        new SamplingConfig(Nearest(TimeSpan.FromMicroseconds(250)));
        // ANCHOR_END: api
    }
}
